clients/sfizz_render.cpp

```diff
@@ -143,7 +143,7 @@ int main(int argc, char** argv)
     ERROR_IF(!synth.loadSfzFile(sfzPath), "There was an error loading the SFZ file.");
     LOG_INFO(synth.getNumRegions() << " regions in the SFZ.");
 
-    fmidi_smf_u midiFile { fmidi_smf_file_read(midiPath.c_str()) };
+    fmidi_smf_u midiFile { fmidi_smf_file_read(midiPath.u8string().c_str()) };
```

Comment:
This is under windows? Should we pass `native()`?

Reply:
The deal here is that Windows native paths are widechar, and fmidi accepts `char` only.

The other problem of Windows is that `char` file APIs use the local ANSI codepage instead of UTF-8. This is always a problem when you'd like to open a file which has extended characters in the path.

The commonly accepted resolution to this, it's that you convert the wide path to UTF-8 (`u8string`), and at app level handle with `MultiByteToWideString` followed by `_wfopen`.
(also it's why I introduced the update at 1226a59)